### ISSUE TITLE
Missing epsilon on SGHMC?

### ISSUE THREAD

**Issue by botev** [hsvgbkhgbv/TACTHMC#1]
Hi,
Thanks to the authors for the nice read. 
I have a question for [these lines](https://github.com/hsvgbkhgbv/TACTHMC/blob/9ab14c436a0708d27dd3f06891227493b9a103b1/sghmc.py#L45) which indicate that you are doing `theta_t = theta_t + v_t` while the correct definition in SGHMC is `theta_t = theta_t + eta_t * v_t` (ignoring the Mass as it seems here its assumed to be unity). 
I have not used Pytorch so maybe I'm missunderstanding something?

**Comment by hsvgbkhgbv** [comment]
Hi,
Thank you for your question.
We fuse eta_t to the line above **the line that you point out**. 
I wish this can answer your question.

**Comment by botev** [comment]
Do you mean that 'module.eta_theta' is somehow the product of the two discretisation constants and the velocity is actually velocity times eta_t?

**Comment by hsvgbkhgbv** [comment]
Yes, it is but this eta_theta may absorb the eta_t that you mentioned. 
When we implement this algorithm, to be consistent to the symbols used for our own method working on sampling parameters for partial observations, we reformulate the hyperparameters. 
If you would like some more details, please read our paper. If you are still doubted, please contact r.luo@cs.ucl.ac.uk for more specific answers.
It that happy?
